src/wallet/rpcwallet.cpp

```diff
@@ -3220,6 +3220,7 @@ static UniValue bumpfee(const JSONRPCRequest& request)
             "                         so the new transaction will not be explicitly bip-125 replaceable (though it may\n"
             "                         still be replaceable in practice, for example if it has unconfirmed ancestors which\n"
             "                         are replaceable)."},
+                            {"change_address", RPCArg::Type::STR_HEX, /* opt */ true, /* default_val */ "", "The change address to pay the fee bump"},
```

Comment:
```suggestion
                            {"change_address", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED_NAMED_ARG, "The change address to pay the fee bump"},
```

Better: Mention what the default/fallback is

Reply:
Done, please review new text.